app/Models/Model.php

```diff
@@ -21,6 +21,9 @@
 namespace App\Models;
 
 use App\Exceptions\ModelNotSavedException;
+use App\Libraries\Transactions\AfterCommit;
```

Comment:
I'd just add an empty method in base model and let the actual models override it instead of having to do the ceremonies...

Reply:
Don't want to be tracking models that don't do anything for after commit